ui/app/src/components/ContentTypeManagement/ContentTypeManagement.tsx

```diff
@@ -61,6 +62,7 @@ export function ContentTypeManagement(props: ContentTypeManagementProps) {
       .subscribe((e: any) => {
         switch (e.data.type) {
           case 'CONTENT_TYPES_ON_SAVED': {
+            dispatch(fetchContentTypes());
```

Comment:
The issue was that when adding a template to the content type and saving, the list of content types were not being updated in the store. So if you wanted to edit the template for the created content it was not going to 'find' any template and then it was going to create a new one based on the content-type id (in this scenario it was the same template name but in lowercase).

Reply:
Make sure TB2 doesn't have this problem.